Riot/Modules/Application/LegacyAppDelegate.m

```diff
@@ -631,6 +653,8 @@ - (void)applicationWillEnterForeground:(UIApplication *)application
     }
     
     _isAppForeground = YES;
+    
+    [self configurePinCodeScreenFor:application createIfRequired:NO];
```

Comment:
Is this call mandatory?
Either it was displayed before in `applicationWillResignActive` or it will be displayed in `applicationDidBecomeActive` on next fging.

Reply:
I've changed the logic here. This is to decide earlier (called before `applicationDidBecomeActive`) whether it's required (after grace time) or not.